README.md

```diff
@@ -180,6 +204,22 @@ output = {
 }
 ```
 
+Add title and description:
+```js
+ms.obj({
+  displayName: 'string',
+}, {title: 'Title', description: 'Desc.'})
+```
+
+Add dependencies:
+```js
+ms.obj({
+  creditCard: 'string',
+  address: 'string'
+}, {dependencies: {creditCard: 'address'}})
```

Comment:
What's your use case for the usage of `dependencies`?
Those are weird examples 😁 

I'd like to have a dependency when a specific property has a certain value, which still won't work. The rest isn't that interesting.

Reply:
I mainly wanted to add `format` to string. But added a bunch more while I was at it. We can leave `dependencies` out of this PR if you prefer. I don't have a use case at the moment.

The example is from over here: https://spacetelescope.github.io/understanding-json-schema/reference/type.html

Reply:
For the use case with a specifc property value I think we have to use `oneOf()` with `const()`. Or is there a better alternative?

```js
{
  oneOf: [ms.obj({
    type: ms.const('a'),
    foo: 'string'
  }), ms.obj({
    type: ms.const('b'),
    bar: 'string'
  })]
}
```

Reply:
no, if you want to cover it with a schema, then there is not.
In our project I've manually handled it using code to get a better error message: https://github.com/upfrontIO/livingdocs-server/blob/master/app/features/menus/menu_api.js#L252-L259

Reply:
Shall I merge?